R/pkg/R/sparkR.R

```diff
@@ -367,19 +367,21 @@ sparkR.session <- function(
     overrideEnvs(sparkConfigMap, paramMap)
   }
   # do not download if it is run in the sparkR shell
-  if (!nzchar(master) || is_master_local(master)) {
-    if (!is_sparkR_shell()) {
-      if (is.na(file.info(sparkHome)$isdir)) {
+  if (!is_sparkR_shell()) {
+    if (!is.na(file.info(sparkHome)$isdir)) {
+      msg <- paste0("Spark package found in SPARK_HOME: ", sparkHome)
+      message(msg)
```

Comment:
Agree that if there exist an existing session, we shouldn't repeat the same message. But it would be useful to have this message for the first time to help debugging. I can imagine users hit issues due to wrong env variables or search path.